src/rancher.js

```diff
@@ -199,10 +229,12 @@ function upgradeAll( http, environment, dockerImage ) {
 			return [];
 		}
 	}
-
-	return environment.listServices()
+   
```

Comment:
@EvanSnapp can we 1) remove the commented out block and 2) try to keep chained called like this broken up so that they're a little easier to read? I realize this is somewhat subjective, but a lot of folks would rather see:

```js
    return call
        .then( ... )
        .then( ... )
        .then( ... );
```
vs
```js
    return call.then( ... ).then( ... ).then( ... );
```